codelists/api.py

```diff
@@ -135,6 +136,7 @@ def codelists_post(request, owner):
         "methodology",
         "references",
         "signoffs",
+        "force_create",
```

Comment:
Should this also be `always_create_new_version`?

Reply:
changed